barretenberg/cpp/src/barretenberg/honk/CMakeLists.txt

```diff
@@ -1,20 +1 @@
-barretenberg_module(honk proof_system commitment_schemes sumcheck)
-
```

Comment:
these lines were causing compilation trouble, established in huddle with Adam that they can be remvoed